lib/expired_ballot_checker.go

```diff
@@ -0,0 +1,77 @@
+package sebak
+
+import (
+	"time"
+	"fmt"
```

Comment:
this one is not used.

Reply:
This issue is worked in progress([WIP]), so it would be better to review after completion for saving your time :)